tests/tensor_search/test_api.py

```diff
@@ -297,4 +298,25 @@ def test_invalid_structured_index_field_features(self):
                 )
                 self.assertEqual(response.status_code, 422)
                 self.assertIn("allFields", response.text)
-                self.assertIn("features", response.text)
\ No newline at end of file
+                self.assertIn("features", response.text)
+
+    def test_log_stack_trace_for_model_properties_not_found(self):
```

Comment:
You can define your tests based on the handlers you have. So something like `test_log_stack_trace_marqo_error`, `test_log_stack_trace_api_error`, `test_log_stack_trace_internal_error` and a fourth one for `Exception` to cover everything not handled